content/articles/common-dns-records.markdown

```diff
@@ -28,9 +28,10 @@ These are the most common DNS record types:
 | [`A` record](/articles/a-record) | This is the most popular type. A records create a DNS record that points to an IPv4 address. It allows you to use memonic names, such as `www.example.com`, in place of IP addresses, like `127.0.0.1`. |
 | [`CNAME` record](/articles/cname-record) | This record works as an alias and maps one name to another. It's often used to reduce duplication in domain name configurations, or to simplify the maintenance of multiple records connected to the same IP address. Its use has increased in the last few years, since it's one of the common mechanisms that's been adopted by cloud services to provision customer-specific services. |
 | [`MX` record](/articles/mx-record) | This record is used to identify the servers mail should be delivered to for a domain. You need to have these records configured to receive emails. |
-| [`TXT` record](/articles/txt-record) | This record allows domain administrators to insert any text content into DNS records. These records are used for various purposes. One example is ownership validation: To prove you own the domain, a provider may require you to add a TXT record with a particular value to your domain. |
+| [`TXT` record](/articles/txt-record) | This record is used to provide to associate text with a domain. |
 | [`NS` record](/articles/ns-record) | This record is used to delegate a subzone to a set of name servers. It's common, because these are the types of records you need to modify when you want to delegate a domain to a DNS provider. |
 | [`SOA` record](/articles/soa-record) | This record stores important information about the DNS zone (your domain). It's common, because each zone must have an SOA record. However, it's unlikely you'll have to create a SOA record directly. For instance, DNSimple automatically manages the SOA records for all your domains. |
+| [`DKIM` record](/articles/dkim-record) | This records stores important information used in the validation of email in transit. |
```

Comment:
DKIM is not a record type. It is a TXT record. There are many other uses for TXT records. As such, a section on the various uses of TXT records would be better.